net/ip.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+/**
+ * Validates whether a given string is a valid IPv4 address
+ *
+ * @param addr IPv4 address in a string format (e.g., "192.168.0.1").
+ * @returns A boolean indicating if the string is a valid IPv4 address
+ *
+ * @example Check if the address is a IPv4
+ * ```ts no-assert ignore
+ * import { isIPv4 } from "@std/net/ip"
+ * import { assert, assertFalse } from "@std/assert"
+ *
+ * const correctIp = "192.168.0.1"
+ * const incorrectIp = "192.168.0.256"
+ *
+ * assert(isIPv4(correctIp))
+ * assertFalse(isIPv4(incorrectIp))
+ *
+ * ```
+ */
+export function isIPv4(addr: string): boolean {
```

Comment:
I wonder if this name confirms to our naming convention. We usually avoid capitalized acronyms (like `IP`). ref https://docs.deno.com/runtime/contributing/style_guide/#naming-convention

Reply:
I thought about this too initially but ended up just thinking this looks fine.